packages/dds/map/src/map.ts

```diff
@@ -297,6 +305,7 @@ export class SharedMap extends SharedObject<ISharedMapEvents> implements IShared
                     blobs.push(blobName);
                     builder.addBlob(blobName, JSON.stringify(headerBlob));
                     headerBlob = {};
+                    totalBlobBytes += currentSize;
```

Comment:
Not sure if I'm missing something but I'm assuming we want to add the totalBlobBytes  outside of the if 

IF currentSize < MaxSnapshotBlobSize  we weill serialize the content at the end but it will still be a blob (line 322)

Reply:
My thinking was that we want this number to hold what we are going to store in the `blobs` property. I don't have any preference on what to do, so I'll probably go for tracking the bytes on all the content objects regardless of if there are included in this `blobs` property or not.